src/navigation/index.jsx

```diff
@@ -18,14 +19,15 @@ export default class Navigation extends Component {
       <NavigationContainer>
         <StatusBar style='auto' />
         <Stack.Navigator
-          initialRouteName='Greeting'
+          initialRouteName='Home'
           screenOptions={{
             headerShown: false,
           }}
         >
           <Stack.Screen name='Greeting' component={Greeting} />
 
           <Stack.Screen name='Login' component={Login} />
+          <Stack.Screen name='Home' component={Home} />
         </Stack.Navigator>
       </NavigationContainer>
```

Comment:
You should improve the navigation, it's better to use a tab navigator if you want a bottom menu for the screens. This allows you to prevent using multiple times the same component for all the screens.

See: 
- [Tab navigation
](https://reactnavigation.org/docs/tab-based-navigation)
- [Screen options with nested navigators](https://reactnavigation.org/docs/screen-options-resolution)

Here is a quick example:
- [WWU navigation](https://github.com/BrightCoders-Institute/BCDIC22-RN-proyecto-final-team3/tree/master/src/navigation)

Reply:
Well check it later, now that the navigations bars are created we will aim towards that change. Thanks for the suggestion.